src/index.ts

```diff
@@ -30,7 +30,7 @@ process.on("SIGUSR2", async () => await cleanupJobResources(jobs));
         .parserConfiguration({"greedy-arrays": false})
         .showHelpOnFail(false)
         .version(packageJson["version"])
-        .wrap(yargs.terminalWidth())
+        .wrap(yargs.terminalWidth?.())
```

Comment:
This makes it possible to run  `gitlab-ci-local` with `bun` to that it's faster to iterate
```
bun ../../gitlab-ci-local/src/index.ts
```